apps/remix-ide/src/app/editor/editor.js

```diff
@@ -150,6 +151,25 @@ class Editor extends Plugin {
       this.currentThemeType = theme.quality
       this.renderComponent()
     })
+    this.on('fileManager', 'currentFileChanged', async (name) => {
+      if (name.endsWith('.ts')) {
+        // extract the import, resolve their content
+        // and add the imported files to Monaco through the `addModel`
+        // so Monaco can provide auto completion
+        let content = await this.call('fileManager', 'readFile', name)
+        const paths = name.split('/')
+        paths.pop()
+        const fromPath = paths.join('/') // get current execution context path
+        for (const match of content.matchAll(/import\s+.*\s+from\s+(?:"(.*?)"|'(.*?)')/g)) {
+          let path = match[2]
+          if (path.startsWith('./') || path.startsWith('../')) path = resolve(fromPath, path)
+          if (path.startsWith('/')) path = path.substring(1)
+          if (!path.endsWith('.ts')) path = path + '.ts'
```

Comment:
This creates an issue for npm module import like ethers and web3 and shows errors:
![Screenshot 2022-05-31 at 12 39 52 PM](https://user-images.githubusercontent.com/30843294/171114281-bf3995d6-140e-4fe4-9641-79f2e861246e.png)

Reply:
done